app/facades/lists/show_facade.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string: true
+
+module Lists
+  class ShowFacade
+    attr_reader :list
+
+    def initialize(list)
+      @list = list
+    end
+
+    def books_list
+      @books = @list.books_lists.map(&:book)
```

Comment:
has_many :books, through: :books_lists -->>>> list.rb
@list.books ->> so you can just use delegate

Reply:
also use getter